src/main/java/uk/gov/companieshouse/filing/processor/FilingProcessorImpl.java

```diff
@@ -63,7 +65,7 @@ private ResponseRecord createResponse(FilingReceived filingReceived) throws Fili
         ResponseRecord response = new ResponseRecord();
         response.setCompanyName(filingReceived.getSubmission().getCompanyName());
         response.setCompanyNumber(filingReceived.getSubmission().getCompanyNumber());
-        String formattedDate = sdk.format(new Date());
+        String formattedDate = sdk.format(dateService.now());
```

Comment:
Does this change really add value?
If the whole format thing was pulled out then we could update to using instants rather than Date, which has kinda been replaced by LocalDate and then Instant.
DateService isn't even being used by other classes.
As it is, was the only purpose of extracting this to give more precise control via mocking during the tests?

Reply:
Yes, the only reason for this extra layer is so that we can test it. 

Good point about the use of Date. Actually, I am currently replacing this to use the new `java.time` objects so that we could use the new thread-safe formatter as well. I'll raise a PR for this shortly